packages/react-ui/src/widgets/legend/LegendIcon.js

```diff
@@ -1,25 +1,16 @@
 import React from 'react';
-import { Box, Grid, makeStyles, Typography } from '@material-ui/core';
+import { Box, Grid, Typography } from '@material-ui/core';
+import PropTypes from 'prop-types';
 
-const useStyles = makeStyles((theme) => ({
-  legendIcon: {
-    alignItems: 'center'
-  }
-}));
-
-export default function LegendIcon({ legend }) {
-  const classes = useStyles();
-
-  if (!legend) {
-    return null;
-  }
+const ICON_SIZE = 16;
 
+function LegendIcon({ legend }) {
   const { labels = [], icons = [] } = legend;
 
   const Icons = labels.map((label, idx) => (
-    <Grid key={label} container item className={classes.legendIcon}>
-      <Box mr={1.5} width={16} height={16}>
-        <img src={icons[idx]} alt={icons[idx]} />
+    <Grid key={label} container item alignItems='center'>
+      <Box mr={1.5} width={ICON_SIZE} height={ICON_SIZE}>
+        <img src={icons[idx]} alt={icons[idx]} width={ICON_SIZE} height={ICON_SIZE} />
```

Comment:
What about just using `alt=""` here, or `label` even it it's redundant? It will be better anyway than the file path or data URL.

Reply:
Yes, label would be much better. I'm change it.